### DISCUSSION TITLE
Il server Discord di Joypad

### DISCUSSION THREAD

**Discussion by sandroli** [pdonadeo/e-uscito-joypad#51] [Announcements]
Ciao a tutti, sono Sandro, ho creato il discord di Joypad.

Nel caso vogliate accesso al discord o smanettare con bot vari ed eventuali rimango a disposizione per darvi i permessi di cui avete bisogno! Scrivete qui o sul discord @rappz 

Grazie per tutto il vostro lavoro!

**Comment by pdonadeo** [comment]
Ciao Sandro, piacere di conoscerti! Bellissima idea quella del canale Discord! Mi mandi il link al canale, così intanto entro e mi presento 😄 

Per bot e cose simili: non ne ho mai fatti per Discord ma... perché no? Vediamo cosa serve e se non porta via troppo tempo.

**Reply by agofa** [reply]
io sono entrato ieri sera, alla chetichella però :-)

**Reply by agofa** [reply]
> Ciao Sandro, piacere di conoscerti! Bellissima idea quella del canale Discord! Mi mandi il link al canale, così intanto entro e mi presento 😄
> 
> Per bot e cose simili: non ne ho mai fatti per Discord ma... perché no? Vediamo cosa serve e se non porta via troppo tempo.

Se non sbaglio è questo
https://discord.gg/avkj2B9M

**Reply by pdonadeo** [reply]
Sono distratto, si vede che me l'ero perso.
